### PR TITLE
Styling/quote boxes

### PR DESCRIPTION
**Pull request by VictoriaFC**
## Summary:
- Updated styling on quote cards and favorites cards to include a quote bubble. 
- Added a favicon
- Added header message to favorites page
- Updated font family for 'Welcome to' on welcome page

## Type of Changes:
- [ ] Bug fix
- [ ] Refactor
- [x] New feature

## How was this tested:
Chrome dev tools

### CODE REVIEW

File: src/FavoriteCard/FavoriteCard.css [hunk 1]
```diff
@@ -1,55 +1,67 @@
...
 	display: flex;
 	flex-direction: column;
   border: 2px solid gray;
-	border-radius: 10px;
+	border-radius: 50%;
 	background-color: white;
-}
-
-.favorite-card-wrapper:hover {
-	transform: scale(1.1);
```
Comment (tfoxcollis) [inline src/FavoriteCard/FavoriteCard.css:10]: Good idea taking the transform off the box!

File: src/QuoteCard/QuoteCard.css [hunk 2]
```diff
@@ -1,59 +1,67 @@
...
-  align-content: stretch;
-  align-items: flex-end;
+	justify-content: center;
+	flex-direction: row;
+	align-items: flex-end;
 }
 
 .quote {
-	font-size: 15px;
+	font-size: small;
```
Comment (tfoxcollis) [inline src/QuoteCard/QuoteCard.css:40]: Ive never used this before! Very cool!

**Review by tfoxcollis — APPROVED** [review]

File: src/App/App.css [hunk 3]
```diff
@@ -1,5 +1,9 @@
-.home-header {
```
Comment (abodnar1) [inline src/App/App.css:1]: Of note: might need to double check that our Cypress test is still passing for this.

Comment (tfoxcollis) [reply src/App/App.css:1]: Good catch @abodnar1!

File: src/FavoriteCard/FavoriteCard.css [hunk 4]
```diff
@@ -1,55 +1,67 @@
...
+	border-radius: 50px;
+	height: 80%;
 }
 
 .delete-button {
 	font-size: 1.5rem;
 	cursor: pointer;
 }
+
+.delete-button:hover {
```
Comment (CorCanavan) [inline src/FavoriteCard/FavoriteCard.css:65]: Love the added transform effects on the icons

File: src/App/App.tsx [hunk 5]
```diff
@@ -24,13 +24,14 @@ const App = () => {
...
 					<Nav />
-						<h2 className="home-header">✨It's time to get Zen in Ten✨</h2>
+						<h2 className="home-page-header">✨It's time to get Zen in Ten✨</h2>
           <Form setAllQuotes={setAllQuotes}/>
           {allQuotes.length > 0 && <Quotes allQuotes={allQuotes} setAllQuotes={setAllQuotes} favList={favList} setFavList={setFavList}/>}
         </Route>
 
         <Route exact path="/favorites" >
 					<Nav />
+					<h2 className="favorite-page-header">Favorites</h2>
```
Comment (CorCanavan) [inline src/App/App.tsx:38]: Yay for the Favorites page header! Just wondering if this should live in the Favorites component?

**Review by CorCanavan — APPROVED** [review]

**Pull request merged**